dynamax/linear_gaussian_ssm/parallel_inference_test.py

```diff
@@ -12,86 +12,108 @@
 from dynamax.linear_gaussian_ssm.inference_test import flatten_diagonal_emission_cov
 
 
-def allclose(x,y, atol=1e-2):
-    m = jnp.abs(jnp.max(x-y))
-    if m > atol:
-        print(m)
-        return False
-    else:
-        return True
-    
+from jax.config import config
+
+config.update("jax_enable_x64", True)
```

Comment:
Tests for marginal likelihood were quite unstable for float32, probably due instability of log det computation. I'd suggest enabling float64 as default for that reason.